examples/group_provisioning_demo.c

```
@@ -0,0 +1,275 @@
+        if (len <= 0) {
+            fprintf(stderr, "[FAIL] Serialize GROUP_JOIN\n");
+            return 1;
+        }
+
+        printf("[Leader] GROUP_JOIN invite sent (%d bytes, encrypted)\n", len);
+    }
+
+    /* Simulate Member A receiving and storing the group key */
+    {
+        uint8_t recv_key[SYMMETRIC_KEY_LENGTH];
+        if (security_get_group_key(GROUP_ID_1, recv_key) != 0) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)